src/python/gem5/prebuilt/hifiveunmatched/hifive-run.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022 The Regents of the University of California
```

Comment:
The "runscript" shouldn't be in `src/`. It should be in `configs/examples` or `configs/known_good`.

Reply:
The location is now configs/example/gem5_library/